app/services/search_query_parser.rb

```diff
@@ -0,0 +1,20 @@
+require 'uri'
+
+class SearchQueryParser
+  API_URL = ENV['TARIFF_QUERY_SEARCH_PARSER_URL']
+
+  def initialize
+    @client = Faraday.new(API_URL) do |conn|
+      conn.request :json
+
+      conn.response :raise_error
+      conn.response :json
+    end
+  end
+
+  def get_tokens(search_query_term)
+    api_response = @client.get('tokens', q: search_query_term)
```

Comment:
Is variable assignment doing much here? Could you just chain the `.body` on the end? eg

```ruby
@client.get('tokens', q: search_query_term)
       .body
```